### ISSUE TITLE
CSS styles not being applied

### ISSUE THREAD

**Issue by manifest3r** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1657]
**OS:** Solus Linux

**Issue Descriptions:** CSS Styles not applying

**Steps to Reproduce:**
Right-Click > Desktop Settings > Apply any theme or specify custom CSS. Styles apply on GPMD settings windows, but not in actual player. Restarting application the loading "circle" page is dark, however once on the main app screen, no CSS styles are applied.
![screenshot from 2016-09-20 09-17-22](https://cloud.githubusercontent.com/assets/10587521/18676619/72b1c104-7f13-11e6-8b07-b406e132f8c9.png)

**Comment by MarshallOfSound** [comment]
I see nothing in this screenshot that indicates that anything is wrong.

I can guarantee that the CSS styles are applied as I personally use them and I know of quite a few others who have played around with them aswell.

I would recommend either providing a CSS stylesheet that "isn't being applied" or familiarizing yourself with CSS rule priority.  You probably just need to add a `!important` somewhere.

**Comment by jostrander** [comment]
One thing I notice is that you're using the same style sheet for both the main window and GPM. If you're CSS isn't targeting GPM specifically then you won't see any changes in the GPM side of things. CSS Styles won't cascade into GPM from the main window as GPM is a webview (similar to an iframe).

**Comment by manifest3r** [comment]
So I first started by applying the provided dark theme, which didn't work. Then I tried the Custom Styles (I pulled the CSS files from https://userstyles.org/styles/115630/play-midnight-for-google-play-music, also attached).

Is there a way to target the webview specifically in the CSS file? It's been a while since I modified CSS.

[css_gpm_rev2.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/483123/css_gpm_rev2.zip)
